data/docs/opentelemetry-collection-agents/opentelemetry-collector/why-to-use-collector.mdx

```diff
@@ -0,0 +1,158 @@
+
+For more examples, you can check guides for scrubbing PII in [logs](https://signoz.io/docs/logs-management/guides/pii-scrubbing/) and [traces](https://signoz.io/docs/traces-management/guides/pii-scrubbing/).
+
+#### Data Transformation
+
+Standardizing telemetry across different environments becomes simpler with the Collector's transformation capabilities. You can normalize attributes names and their values to make consistent telemetry data across different services and teams with the [transform processor](https://signoz.io/docs/opentelemetry-collection-agents/opentelemetry-collector/configuration/#transform-processor). The Collector also supports converting between different data formats, aggregating metrics before export, and enriching telemetry data with additional contextual information.
+
+### 2. Auto-Added Resource Attributes
+
+The Collector automatically enhances your telemetry data with additional context information about the infrastructure and environment where your application runs.
+
+#### Infrastructure Context
+
+Without any manual configuration, the Collector adds metadata including hostname, operating system, and other details about the environment. In cloud environments it automatically detects and adds provider specific metadata such as AWS instance IDs, GCP project information, Azure resource details, along with region and availability zone data.
+
+For containerized applications, especially in Kubernetes, the Collector adds metadata with pod names, namespaces and container IDs.
+
+Check out these guides for setting resource attributes using [resource processor](https://signoz.io/docs/logs-management/guides/set-resource-attributes-for-logs/#using-resource-processor) and [resource detection processor](https://signoz.io/docs/logs-management/guides/set-resource-attributes-for-logs/#using-resource-detection-processor).
+
+#### Benefits for Legacy Systems
+
+You can add rich observability context without touching the legacy codebase. The Collector ensures unified tagging and consistent resource attributes across all services, regardless of their technology stack.
+
+This standardization enables you to have better correlation of telemetry data from different sources and simplifies debugging by making it clear which host, container, or cloud instance generated telemetry data.
+
+### 3. Export to Multiple Destinations
+
+The OTel Collector supports exporting to multiple destinations simultaneously, which is very helpful during migrations. This capability allows you to gradually migrate from existing observability tools to SigNoz.
```

Comment:
Mention S3 exporter and Add link to https://signoz.io/docs/tutorial/infinite-retention-aws-s3/